drivers/gps.py

```diff
@@ -109,11 +108,11 @@ def write(self, stream_id, data):
         from drivers.logserial import LogSerial
         from drivers.bus import BusHandler
 
-        config_serial = {'port': 'COM5', 'speed': 4800, 'stream_id': 1}
-        config_gps = {'stream_id': 2}
+        config_serial = {'port': 'COM5', 'speed': 4800}
+        config_gps = {}
         log = LogWriterEx(prefix='gps-test')
-        device = GPS(config_gps, bus=BusHandler(log, out={2:[]}))
-        device0 = LogSerial(config_serial, bus=BusHandler(log, out={1:[(device.bus.queue, 1)]}))
+        device = GPS(config_gps, bus=BusHandler(log, name='gps', out={'position':[]}))
+        device0 = LogSerial(config_serial, bus=BusHandler(log, name='serial1', out={'raw':[(device.bus.queue, 'nmea')]}))
```

Comment:
Could we use `gps` and `serial` here instead of `device` and `device0`?

Reply:
`LogSerial` outputs stream of bytes, right? It does not output nmea sentences/packet. If this is so, I would not name the input on `GPS` `nmea` if it expects raw bytes possibly intermixed with binary gps protocol.